### PR TITLE
blender-fix: ceil floats to avoid hashing issues

### PR DESCRIPTION
**Pull request by schinkowski**
Round normals to 2 decimal places to avoid issues with meshes being sent every frame

### CODE REVIEW

**Comment by sindharta** [comment]
We would need to get https://github.com/Unity-Technologies/MeshSyncDCCPlugins/pull/178 merged first before reviewing this.

File: Plugins~/Src/MeshSyncClientBlender/msblenContext.cpp [hunk 1]
```diff
@@ -905,8 +905,9 @@ void msblenContext::doExtractNonEditMeshData(msblenContextState& state, BlenderS
         blender::barray_range<mu::tvec3<float>> normals = bmesh.normals();
         if (!normals.empty()) {
             dst.normals.resize_discard(num_indices);
+            const float roundingFactor = 100.0f;
             for (size_t ii = 0; ii < num_indices; ++ii)
-                dst.normals[ii] = normals[ii];
+                dst.normals[ii] = ceil(normals[ii] * roundingFactor) / roundingFactor;
```
Comment (SonnyCampbellUnity) [inline Plugins~/Src/MeshSyncClientBlender/msblenContext.cpp]: Can you create a utility function for this in the msblenUtils class or something like that? Seems like this logic would be commonly re-used

Comment (schinkowski) [reply Plugins~/Src/MeshSyncClientBlender/msblenContext.cpp]: Done here https://github.com/unity3d-jp/MeshSync/pull/710

File: Plugins~/Src/MeshSyncClientBlender/msblenContext.cpp [hunk 2]
```diff
@@ -901,7 +901,7 @@ void msblenContext::doExtractNonEditMeshData(msblenContextState& state, BlenderS
         if (!normals.empty()) {
             dst.normals.resize_discard(num_indices);
             for (size_t ii = 0; ii < num_indices; ++ii)
-                dst.normals[ii] = normals[ii];
+                dst.normals[ii] = ms::roundToDecimals(normals[ii]);
```
Comment (schinkowski) [inline Plugins~/Src/MeshSyncClientBlender/msblenContext.cpp]: Also needs this to work: https://github.com/unity3d-jp/MeshSync/pull/710/files

**Review by SonnyCampbellUnity — APPROVED** [review]

**Review by seandillon92 — APPROVED** [review]

**Pull request merged**